example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -398,15 +398,26 @@ describe('The S3 Ingest Granules workflow', () => {
       expect(resourceURLs.includes(s3CredsUrl)).toBe(true);
     });
 
-    it('updates the CMR metadata "online resources" with the proper types', () => {
+    it('updates the CMR metadata "online resources" with the proper types and urls', () => {
       const resource = ummCmrResource;
-      const expected = [
+      const distributionUrl = getDistributionFileUrl({
+        bucket: files[0].bucket,
+        key: files[0].filepath
+      });
+      const s3Url = getPublicS3FileUrl({ bucket: files[2].bucket, key: files[2].filepath });
+      const s3CredsUrl = resolve(process.env.DISTRIBUTION_ENDPOINT, 's3credentials');
+      const expectedTypes = [
         'GET DATA',
         'VIEW RELATED INFORMATION',
         'VIEW RELATED INFORMATION',
         'GET RELATED VISUALIZATION'
       ];
-      expect(expected).toEqual(resource.map((r) => r.Type));
+      const cmrUrls = resource.map((r) => r.URL);
+
+      expect(cmrUrls.includes(distributionUrl)).toBe(true);
```

Comment:
It seems like lines 417 - 419 are kind of redundant with https://github.com/nasa/cumulus/pull/894/files#diff-45ed34e4b37c728d5f2e32f579c4cfc9R385 ?

Reply:
These are testing the URLs in `ummCmrResource` and the test on line 385 is testing the URLs in `cmrResource`.
The difference between those is 
`getOnlineResources(granule),`
`getOnlineResources(ummGranule),`

I'm not completely sure what the difference is but I wanted to cover the bases

Reply:
Ah, you're right. It looks like the tests above are for the online resources from the ECHO10 XML metadata, but this test is verifying that the online resources are also correct when you request the CMR metadata as UMM-G. @ifestus is that right?